### PR TITLE
Propagate the last year of changes into the documentation.

### LINKED ISSUE #7251: dealii claims copyright for future years
When I built dealii in the year 2033, I got such a diff:

```diff
--- /usr/share/doc/packages/dealii/doxygen/deal.II/work__stream_8h_source.html      2033-10-28
+++ /usr/share/doc/packages/dealii/doxygen/deal.II/work__stream_8h_source.html      2018-09-28
@@ -18,7 +18,7 @@
 <link rel="SHORTCUT ICON" href="deal.ico"></link>
 <script type="text/javascript" src="custom.js"></script>
 <meta name="author" content="The deal.II Authors <authors@dealii.org>"></meta>
-<meta name="copyright" content="Copyright (C) 1998 - 2018 by the deal.II authors"></meta>
+<meta name="copyright" content="Copyright (C) 1998 - 2033 by the deal.II authors"></meta>
 <meta name="deal.II-version" content="8.5.1"></meta>
 </head>
```

See also https://reproducible-builds.org/ for why this matters.

This bug was found while working on reproducible builds for openSUSE.

### PR DESCRIPTION
**Pull request by bangerth**
Follow-up to #7257. Fixes #7251.

### CODE REVIEW

File: doc/doxygen/CMakeLists.txt [hunk 1]
```diff
@@ -136,9 +125,24 @@ FOREACH (_source ${_source_files})
...
+  )
+
+CONFIGURE_FILE(
+  ${CMAKE_CURRENT_SOURCE_DIR}/scripts/mod_footer.pl.in
+  ${CMAKE_CURRENT_BINARY_DIR}/scripts/mod_footer.pl
+  )
+
+
+
+
```
Comment (tamiko) [inline doc/doxygen/CMakeLists.txt]: What about we only use 2 or 3 newlines here? :smile:

Comment (bangerth) [reply doc/doxygen/CMakeLists.txt]: Sure, if that what you'd like :-)

**Comment by bangerth** [comment]
Patch is updated.

**Comment by bangerth** [comment]
I fixed this in essentially the way you suggest in the second commit here.

**Comment by bmwiedemann** [comment]
I only see 1 commit in this PR. Did you push it to the right place?

**Comment by Rombur** [comment]
@bangerth it's time for a coffee break :D

**Comment by bangerth** [comment]
@Rombur -- you don't know how right you actually are. I was at a concert last night with @marcfehling last night where the main act came on stage from 11:30pm to 1am. I was home at 2:30am. I got up at 6:30 so that I could be in my office at 7:30 where I have been participating in ECP project reviews every since. I *do* *definitely* *need* *coffee* :-)

In any case, the commit is now here.

**Pull request merged**